notes.go

```diff
@@ -53,6 +53,7 @@ type Note struct {
 	ExpiresAt    *time.Time    `json:"expires_at"`
 	UpdatedAt    *time.Time    `json:"updated_at"`
 	CreatedAt    *time.Time    `json:"created_at"`
+	ResolvedAt   *time.Time    `json:"resolved_at"`
```

Comment:
Please place the field below the `ResolvedBy` field, following the ordering in the docs.